apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/PartitionedReadTableFixture.cs

```diff
@@ -42,42 +45,41 @@ protected override void CreateTable()
 
         protected override void PopulateTable(bool fresh)
         {
+            using var connection = GetConnection();
+            connection.Open();
+
             // Assume old data is still valid
-            if (!fresh)
+            if (fresh)
             {
-                return;
-            }
-            const int PartitionSize = 5000;
+                const int PartitionSize = 5000;
```

Comment:
Possibly move this bit into a local method, so that in the non-fresh case we don't need to read past the large block? (Not a big deal though.)